pom.xml

```diff
@@ -189,6 +189,16 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>jackson2-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.pivovarit</groupId>
+            <artifactId>parallel-collectors</artifactId>
+            <version>2.5.0</version>
+        </dependency>
+        <dependency>
+            <groupId>ca.umontreal.iro.simul</groupId>
+            <artifactId>ssj</artifactId>
+            <version>3.3.1</version>
```

Comment:
The license is fine although the version of gson it uses is vulnerable, gson would need to be excluded from the dependency.

If gson is actually needed then it should be replaced by the gson API plugin:
https://plugins.jenkins.io/gson-api/dependencies/

Reply:
Reverted.